metpy/calc/thermo.py

```diff
@@ -740,3 +740,70 @@ def psychrometric_vapor_pressure_wet(dry_bulb_temperature, wet_bulb_temperature,
     """
     return (saturation_vapor_pressure(wet_bulb_temperature) - psychrometer_coefficient *
             pressure * (dry_bulb_temperature - wet_bulb_temperature).to('kelvin'))
+
+
+def pressure_at_height_above_pressure(sfc_pressure, height):
+    """
+    Calculate the pressure of a certain height above another pressure level.
+
+    Assumes a standard atmosphere.
+    """
+    sfc_height = pressure_to_height(sfc_pressure)
+    AGL_height = sfc_height + height
+    return pressure_at_height_above_pressure(AGL_height)
+
+
+def mixed_layer(p, variable):
```

Comment:
I'm thinking that the depth, conversions, masking, interpolation, etc all goes here.